src/main/scala/io/iohk/ethereum/domain/Blockchain.scala

```diff
@@ -107,6 +107,19 @@ trait Blockchain {
       ethCompatibleStorage: Boolean
   ): StorageProof
 
+  /**
+    * Get the MptStorage
+    * @param blockNumber
+    * @return MptStorage
+    */
+  def getBackingStorage(blockNumber: BigInt): MptStorage
```

Comment:
why not to name it `getBackingMptStorage` and `getReadOnlyMptStorage`? otherwise it's to generic

Reply:
Sure, I just added the original name, but this is better. Done